cli/flags.rs

```diff
@@ -555,6 +558,12 @@ fn fmt_subcommand<'a, 'b>() -> App<'a, 'b> {
         .help("Check if the source files are formatted.")
         .takes_value(false),
     )
+    .arg(
+      Arg::with_name("stdin")
+        .long("stdin")
+        .help("Read the code from stdin.")
```

Comment:
It looks like besides reading from stdin it outputs code to stdout. Can you describe this behavior in help text? (Maybe add an example to subcommand as well)

Reply:
This is another reason `-` would be better. The meaning of this functionality is specifically "`stdin`/`stdout`". This is conventionally represented with `-`, whereas `--stdin` is misleading. I feel more strongly about it now.

Reply:
@nayeemrmn `-` can be a valid filename. (nah I'm just joking + bikeshedding here)

Yeah `--stdin` is a bit long to type and its implicit effect on stdout is definitely slightly confusing... If @bartlomieju also thinks `-` is better then it is a very easy change to make.

Reply:
Yeah, SGTM, if that's an easy change I guess it makes more sense - we already have similar argument in `catj`.